tests/integration/cli/test_custom_module.py

```diff
@@ -46,7 +46,7 @@ def test_ssh_regular_module(self):
         Test regular module work using SSHCase environment
         """
         expected = "hello"
-        cmd = self.run_function("test.echo", arg=["hello"])
+        cmd = self.run_function("test.echo", args=["hello"])
```

Comment:
I just checked the `run_function` and it looks to be expecting `arg`. Can you elaborate why you are changing this on this line and the ones below

Reply:
## arg
The` run_function` method might expect a single argument and uses the parameter name `arg`. In this case, the first line `(arg=["hello"])` would be the correct usage.

## args
The` run_function` method might expect multiple arguments and uses the parameter name` args`. In this case, the second line `(args=["hello"])` would be the correct usage.